js/main.js

```diff
@@ -1,3 +1,24 @@
+var tabs = document.querySelectorAll('.tab');
+var allPanels = document.querySelectorAll(".schedule");
+
+function activateTab(tab) {
+  //remove active de todas as outras tabs
+  for (i = 0; i < tabs.length; ++i) {
```

Comment:
here you can use a `forEach` ousted a `for` https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
https://clovisdasilvaneto.github.io/explorando-javascript-filter-reduce-map-every-some-e-foreach/